sklearn/linear_model/tests/test_ridge.py

```diff
@@ -704,6 +706,36 @@ def test_sparse_design_with_sample_weights():
                                       decimal=6)
 
 
+def test_ridgecv_int_alphas():
+
+    X = np.array([[-1.0, -1.0], [-1.0, 0], [-.8, -1.0],
+                  [1.0, 1.0], [1.0, 0.0]])
+    y = [1, 1, 1, -1, -1]
+
+    # Integers
+    ridge = RidgeCV(alphas=(1, 10, 100))
+    ridge.fit(X, y)
+
+
+def test_ridgecv_negative_alphas():
+
+    X = np.array([[-1.0, -1.0], [-1.0, 0], [-.8, -1.0],
+                  [1.0, 1.0], [1.0, 0.0]])
+    y = [1, 1, 1, -1, -1]
+
+    # Negative integers
+    ridge = RidgeCV(alphas=(-1, -10, -100))
+    assert_raises_regex(ValueError,
+                        "alphas cannot be negative.",
+                        ridge.fit, X, y)
+
+    # Negative alphas
```

Comment:
I think this test is redundant. We don't need too much tests for such a minor issue.

Reply:
I added that test because the initial error stated: `ValueError: Integers to negative integer powers are not allowed.` So I wanted to double check that both negative integers and floats work.

Reply:
You don't persuade me here but I won't focus too much on that.
I just think such a minor thing doesn't deserve so many tests.